### PR TITLE
Expose extensibility point for FieldsChangeService

### PR DESCRIPTION
**Pull request by belchevgb**

### CODE REVIEW

File: fields-change/news-field-change.service.ts [hunk 1]
```diff
@@ -0,0 +1,27 @@
+import { FieldsChangeService, FieldWrapper, FIELDS_CHANGE_SERVICE_TOKEN } from "progress-sitefinity-adminapp-sdk/app/api/v1";
+import { Injectable } from "@angular/core";
+
+const NEWS_TYPE_FULL_NAME = "Telerik.Sitefinity.News.Model.NewsItem";
+const TITLE_FIELD_NAME = "Title";
+const RITCH_TEXT_EDITOR_FIELD_NAME = "Content";
+
+@Injectable()
+export class NewsFieldsChangeService implements FieldsChangeService {
```
Comment (mstratiev) [inline fields-change/news-field-change.service.ts:9]: name it mb something like CustomFieldsChangeService?

Comment (mstratiev) [reply fields-change/news-field-change.service.ts:9]: and leave js doc around here which is which and what it does

Comment (belchevgb) [reply fields-change/news-field-change.service.ts:9]: But it's not for custom fields. It's for all fields in the form.

File: fields-change/news-field-change.service.ts [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+ * Service which listens for field changes for news items.
+ *
+ * @export
+ * @class NewsFieldsChangeService
+ * @implements {FieldChangeService}
+ */
+@Injectable()
+export class NewsFieldsChangeService implements FieldChangeService {
+    /**
+     * Method called when on of the field changes.
```
Comment (Novkirishki) [inline fields-change/news-field-change.service.ts:18]: Method called on every field change

File: fields-change/news-field-change.service.ts [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+
+            ritchTextEditorField.writeValue(changedValue);
+        }
+    }
+
+    /**
+     * Determines if this service can process the changed field.
+     *
+     * @param {string} typeFullName The CLR full name of the item(ex: Telerik.Sitefinity.News.Model.NewsItem)
+     * @returns {boolean} Wheter the field can be processed.
```
Comment (Novkirishki) [inline fields-change/news-field-change.service.ts:38]: A value indicating whether the service can handle field changes.

File: fields-change/news-field-change.service.ts [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+        // When the title field is changed, we want to update the long text field with the same value(mirroring).
+        if (changedFieldName === TITLE_FIELD_NAME) {
+            const ritchTextEditorField = fields.find(x => x.fieldModel.key === RITCH_TEXT_EDITOR_FIELD_NAME);
+
+            ritchTextEditorField.writeValue(changedValue);
+        }
+    }
+
+    /**
+     * Determines if this service can process the changed field.
```
Comment (Novkirishki) [inline fields-change/news-field-change.service.ts:35]: Determines whether the service can process a field change.

File: fields-change/news-field-change.service.ts (hunk 1 above)
Comment (Novkirishki) [reply fields-change/news-field-change.service.ts]: agree with Marto here. Custom shows that the service is custom and not that it is related to custom fields

**Pull request merged**